content/en/feature_flags/setup/ruby.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Ruby Feature Flags Support
+description: Set up Datadog Feature Flags for Ruby applications.
+further_reading:
+- link: "/feature_flags/setup/"
+  tag: "Documentation"
+  text: "Feature Flags Setup"
+- link: "/apm/"
+  tag: "Documentation"
+  text: "APM Setup"
+---
+
+## Prerequisites
+
+Before setting up the Ruby Feature Flags SDK, ensure you have:
+
+- **Datadog Agent** with [Remote Configuration][1] enabled
+- **Datadog Ruby tracer** `datadog` version 2.23.0 or later
+- **OpenFeature Ruby SDK** `openfeature-sdk` version 0.4.1 or later
+- **Service and environment configured** - Feature flags are targeted by service and environment
+
+## Installing and initializing
+
+Feature Flagging is provided by APM. To integrate APM into your application with feature flagging support, install the required gems and configure remote configuration with OpenFeature support.
+
+```shell
+gem install ddtrace openfeature-sdk
+```
+
+```ruby
+require 'datadog'
+require 'open_feature/sdk'
+require 'datadog/open_feature/provider'
+
+# Configure Datadog with feature flagging enabled
+Datadog.configure do |config|
+  config.remote.enabled = true
+  config.open_feature.enabled = true
+end
+
+# Configure OpenFeature SDK with Datadog provider
+OpenFeature::SDK.configure do |config|
+  config.set_provider(Datadog::OpenFeature::Provider.new)
+end
+
+# Create OpenFeature client
+client = OpenFeature::SDK.build_client
+```
+
+The client returns default values until remote configuration loads in the background. This approach keeps your application responsive during startup but may serve defaults for early requests.
```

Comment:
Provider eventing is not yet supported in OpenFeature: https://github.com/open-feature/ruby-sdk?tab=readme-ov-file#eventing

Eventing is needed for proper separation of non-blocking-vs-blocking between `set_provider` vs `set_provider_and_wait` so I'm leaving that out of the docs for now. I'm currently working on a pull request to get OpenFeature's `set_provider_and_wait` to match the event-based OpenFeature spec and then I can add a section for blocking to this doc